tests/test_interface_private_vlan.rb

```diff
@@ -273,10 +273,8 @@ def test_interface_switchport_pvlan_host_assoc_default
       return
     end
 
-    input = []
-    assert_equal(input,
-                 interface.switchport_mode_private_vlan_host_association,
-                 'Err: switchport private host_association not configured')
+    refute(interface.switchport_mode_private_vlan_host_association,
```

Comment:
I'm not sure you should have changed the assert_equal for these array tests. 

`refute` is testing for false, not empty array. You can use `refute_empty?` to fail if it's not an empty array. 

Be sure to retest after making these changes.